src/govuk/components/radios/__snapshots__/template.test.js.snap

```diff
@@ -2,7 +2,7 @@
 
 exports[`Radios nested dependant components passes through fieldset params without breaking 1`] = `
 <fieldset class="govuk-fieldset app-fieldset--custom-modifier"
-          aria-describedby="example-hint example-error"
+          aria-describedby="undefined-hint"
```

Comment:
Is `undefined-hint` the correct value here?

Reply:
Good spot 🙂

Reply:
Fixed https://github.com/alphagov/govuk-frontend/pull/1924/commits/e50c84d004a7efec453ef7aaa4023a57f6528716#diff-3109d45dad6a798504c2feec8b758819R5